### ISSUE TITLE
Issue: blender eats more than just fruit

### ISSUE THREAD

**Issue by Slayerhero90** [naelstrof/KoboldKare#126]
I was attempting to position a wide bucket/trough in front of a countertop blender. I definitely had this happen in the kitchen; don't know for sure if I saw it in the starting house too. Unfortunately, the traction effect that pulls fruit to the top of the blender grabbed the empty trough and pulled it into the blender, destroying it completely and (fairly) yielding no reagents.

**Comment by naelstrof** [comment]
Yeah, while blending buckets is intentional, I could be more stringent with the detection so it doesn't suck in empty buckets.
